backend/src/main/java/com/messenger/backend/services/UserService.java

```diff
@@ -13,25 +16,30 @@ public class UserService {
     public UserService(UserRepository userRepository) {
         this.userRepository = userRepository;
     }
+
     public UserEntity getByUserName(String userName) {
         return userRepository.findByUserName(userName);
     }
 
     public Boolean getUserStatus(Integer id) {
-        Boolean status =false;
         UserEntity user = userRepository.findAllById(id);
-        status = user.getActiveStatus();
+        boolean status = user.getActiveStatus();
         return status;
     }
 
+    public UserEntity createUser(String userName) {
+        UserEntity newUser = new UserEntity(null, userName, new Date(), true, Collections.emptySet());
+        return userRepository.save(newUser);
+    }
 
     public UserEntity updateContactList(UserEntity userEntity) {
         Integer id = userEntity.getId();
+        UserEntity result = UserEntity.EMPTY_ENTITY;
```

Comment:
so when userRepository.existsById(id)=null we return EMPTY_ENTITY and make also return status code 404? Or just one of these?